bc_obps/registration/api.py

```diff
@@ -19,10 +19,32 @@
     UserOperatorOut,
 )
 from registration.utils import check_users_admin_request_eligibility, update_model_instance
-
+from .models import Operation, Operator, NaicsCode, NaicsCategory, User
+from decimal import *
+from uuid import *
+from django.core.management import call_command
+from django.http import JsonResponse
+from django.conf import settings
+from django.http import HttpResponse
+import os
 
 router = Router()
 
+# testing endpoint
+@router.get("/test-setup")
+def setup(request):
+    print(settings.ENVIRONMENT)
+    print(type(settings.ENVIRONMENT))
```

Comment:
do we still need these print statements? It's not a big deal now, but we'll want to make sure they're removed before we go to production